### PR TITLE
NS-845: Adds job queue for Canvas API jobs

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/NS-845

### CODE REVIEW

**Comment by johncrossman** [comment]
Looks good to me. @sandeepmjay - You want to review?

File: nessie/lib/metadata.py [hunk 1]
```diff
@@ -30,6 +30,17 @@
 from nessie.externals import rds, s3
 
 
+def create_canvas_api_import_status(job_id, term_id, course_id, table_name):
+    sql = f"""INSERT INTO {_rds_schema()}.canvas_api_import_job_status
+               (job_id, term_id, course_id, table_name, status, instance_id, created_at, updated_at)
```
Comment (sandeepmjay) [inline nessie/lib/metadata.py:35]: @lyttam Does It make sense to include a column on what type of Canvas API we are importing from ?

**Comment by sandeepmjay** [comment]
Looks good @lyttam. I had posted my thoughts on a minor addition to the import status table. I can merge once you have taken a look. Thanks!

File: tests/conftest.py [hunk 2]
```diff
@@ -84,6 +84,18 @@ def metadata_db(app):
         created_at TIMESTAMP NOT NULL,
         updated_at TIMESTAMP NOT NULL
     )""")
+    rds.execute(f"""CREATE TABLE IF NOT EXISTS {rds_schema}.canvas_api_import_job_status
+    (
+        job_id VARCHAR NOT NULL,
+        course_id VARCHAR NOT NULL,
+        term_id VARCHAR NOT NULL,
+        table_name VARCHAR NOT NULL,
```
Comment (sandeepmjay) [inline tests/conftest.py:92]: Same question as above, including the API we are interacting with, might make this status table general purpose for any of the future API import requirements as well?

File: nessie/lib/metadata.py (hunk 1 above)
Comment (lyttam) [reply nessie/lib/metadata.py:35]: @sandeepmjay the `table_name` column will contain values such as 'gradebook_history' and 'grade_change_log' (referencing the tables in the canvas_api_ext_dev schema, but also identifying which API was used).

Comment (sandeepmjay) [reply nessie/lib/metadata.py:35]: sounds good. Thanks, @lyttam.

**Pull request merged**